mautic/mautic
# Fix the typos in php doc

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [yes]
| New feature/enhancement? (use the a.x branch)      | [no]
| Deprecations?                          | [no]
| BC breaks? (use the c.x branch)        | [no]
| Automated tests included?              | [no] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



This PR fixes the typos in php docs. These typos affect phpstan analysis.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. There is no change in functionality, so just make sure the tests work well and code review




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11312"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
